README.md

```diff
@@ -24,10 +24,10 @@ PhoneGap plugin for managing application assets with javascript asset maps. Incl
 
 ### downloadFile()
 
-**wizAssets.downloadFile(String URL, String filePathToBeStoredWithFilename, Function success, Function fail);**
+**wizAssets.downloadFile(String URL, String URI, Function success, Function fail);**
```

Comment:
This URI is just a name for a resource. It doesn't have to be a path, it doesn't have to be a relative path and it might even break if you were to use a full path.
All the users have to know is that if they use a file path-like name, folders hierarchy will be automatically created.